src/main/resources/config/application-buildagent.yml

```diff
@@ -8,8 +8,8 @@ artemis:
     repo-download-clone-path: ./repos-download
     version-control:
         url: http://localhost:8000  # TODO: is this really needed for build agents?
-        user: artemis_admin # Required for https access to localvc
-        password: artemis_admin # Required for https access to localvc. You can otherwise use a token, or ssh key
+        build-agent-git-username: buildjob_user # Required for https access to localvc
```

Comment:
add a comment that admins must change this, otherwise there is some security risk here. Please add this into the docs as well